CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 
 # CMake file to compile this BabbleSim component as a west module in Zephyr
 
-if(CONFIG_BOARD_NRF52_BSIM)
+if(CONFIG_BOARD_NRF52_BSIM OR CONFIG_BOARD_NRF53_BSIM)
```

Comment:
Maybe we should add a configuration `CONFIG_BOARD_BSIM_COMPATIBLE` in upstream zephyr so that it becomes easier to support other boards in the future?